cmd/authenticator/main.go

```diff
@@ -74,3 +76,7 @@ func printErrorAndExit(errorMessage string) {
 	errLogger.Printf(errorMessage)
 	os.Exit(1)
 }
+
+func printVersion(){
+	infoLogger.Printf("Kubernetes Authenticator Client v%s starting up...", authenticator.FullVersionName)
```

Comment:
Conjur Kubernetes Authenticator Client?

Reply:
Do we need the full name though? Seems like "Authenticator Client" should be enough to identify it probably.

Reply:
@sigalsax move to message to `log_messages.go` and add to it prefix `CAKC` like all other messages

Reply:
Wanted to avoid saying "Conjur"-type naming to avoid deciding when to use DAP and when Conjur. Updating to Authenticator Client and moving this to `log_messages.go`

Reply:
I would actually go with `Kubernetes Authenticator Client` as it's the name in the docs. Sorry for the ping pong 🏓